.changeset/flat-guests-marry.md

```diff
@@ -0,0 +1,6 @@
+---
+"chainlink": minor
+---
+
+#changed Gas Estimator L1Oracles to be chain specific
```

Comment:
First time seeing these tags but think we should just mark this as `#internal` since the functionality should still be the same